### ISSUE TITLE
several group id

### ISSUE THREAD

**Issue by IlyaEremin** [yigit/android-priority-jobqueue#184]
How can I manage next situation:
I have two queue with different groupId, let's call them `queue1` and `queue2`. They should run in parallel. Before I can run any of these queues I need to finish task `A`.
I can't figure out how I can achieve this with the library.
If there is an ability to add several groupId to one job so one job with highest priority can blocks several queue with different groupId.
possible duplicate of #136

**Comment by yigit** [comment]
Unfortunately, there is no clear way to do this right now. You can make the first job add the others but I understand it is not great.
Post v2.0, I'll planning to work on some way to chain jobs.

**Comment by IlyaEremin** [comment]
![okay](https://cloud.githubusercontent.com/assets/1190759/16146385/6441718c-3486-11e6-87ab-3a57a5af4eab.png)
